test/formatters/formatters.outputBlockFormatter.js

```diff
@@ -1,5 +1,6 @@
-var assert = require('assert');
-var formatters = require('../lib/web3/formatters.js');
+var assert   = require('assert');
+//var assert = chai.assert;
```

Comment:
try not to add commented out code. also, I would like us to move from node assert to chai assert.